plato/domain.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class StorageType(str, Enum):
```

Comment:
I don't think that it is worth it to create a domain module for this project, especially since this is more of a development/setup thing and not a "Plato domain" thing. Perhaps it can be defined directly in the file_storage file?

Reply:
Maybe I was too overzealous 💅 I'll change it